src/features/courses/course-route.ts

```diff
@@ -361,6 +385,27 @@ router.put('/question/grade/:id',
         }));
     }));
 
+router.put('/question/extend',
+authenticationMiddleware,
+validate(extendCourseTopicQuestionValidation),
+asyncHandler(async (req: RederlyExpressRequest<any, ExtendCourseTopicQuestionRequest.body, unknown, any, unknown>, _res: Response, next: NextFunction) => {
+    const query = req.query as ExtendCourseTopicQuestionRequest.query;
+    const body = req.body as ExtendCourseTopicQuestionRequest.body;
+    try {
```

Comment:
We have this mistake a few different places in the application
We don't need a try catch because `asyncHandler` does that for us

Reply:
Nit ^